src/category_theory/groupoid.lean

```diff
@@ -6,11 +6,13 @@ Authors: Reid Barton
 
 import category_theory.category
 import category_theory.isomorphism
+import data.equiv.basic
 
 namespace category_theory
 
 universes v u -- declare the `v`'s first; see `category_theory.category` for an explanation
 
+/-- A `groupoid` is a category such that all homomorphismss are isomorphisms. -/
```

Comment:
I would just say "morphisms" for a general category, not "homomorphisms".

Reply:
Also, the word ends in double-s, currently.

Reply:
```suggestion
/-- A `groupoid` is a category such that all morphisms are isomorphisms. -/
```